Light_Install.rst

```diff
@@ -0,0 +1,68 @@
+**लाइट इनस्टॉल गाइड**
+
+मैटरमोस्ट एक मैसेजिंग और कोलैबोरेशन मंच है। मैटरमोस्ट के द्वारा,आप प्रतिदिन इस्तेमाल किए जाने वाले उपकरण एक ही जगह पर एकीकृत कर सकते है और कोई भी  अधिसूचना या कार्य नहीं छूटेगा।
+
+आरंभ करने से पहले आपको इनकी आवश्यकता होगी:
+- रूट लेवल एक्सेस वाला उबंटू सर्वर (18.04 या 20.04)
+- (रेकमेंडेड) आपके सर्वर से जुड़ा एक डोमेन नाम (यानी mymattermostserver.com)
```

Comment:
Shouldn't "recommended" be translated? Perhaps "anushashit" like you did below?

Also, yani doesn't mean e.g. Perhaps "jaise ki" makes more sense?

Reply:
Although it means "that is" and is referring to a particular link, not any example in this case, still done the change.